src/main/java/net/dv8tion/jda/api/events/thread/member/ThreadMemberLeaveEvent.java

```diff
@@ -17,12 +17,18 @@
 package net.dv8tion.jda.api.events.thread.member;
 
 import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Member;
 import net.dv8tion.jda.api.entities.ThreadMember;
 import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
 
 import javax.annotation.Nonnull;
 
-//TODO-v5: Docs
+/**
+ * Indicates that a {@link Member GuildMember} left a {@link ThreadChannel}.
```

Comment:
Please stop using "GuildMember" everywhere, it looks like an actual type in the docs if you read this and only confuses the reader more.